src/main.rs

```diff
@@ -1,87 +1,75 @@
-#![allow(dead_code, unused_variables, unused_imports)]
-pub mod accelerator;
+pub mod driver;
 pub mod motions;
 pub mod operation;
 pub mod drivers;
-#[cfg(test)]
-mod tests;
 
 #[macro_use]
 extern crate clap;
-use clap::{App, SubCommand, ArgMatches};
-use std::io::*;
-use std::process;
 
-use drivers::Driver;
+use clap::{App, Arg, SubCommand, ArgMatches, AppSettings};
+use std::io::prelude::*;
+use std::io::stdin;
+use std::process;
+use driver::Driver;
```

Comment:
I'm not convinced this repetition is a bad thing or even worse than `drivers::Driver`.